spec/factories/responses.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :response do
+    text "MyString"
+    created_at "2016-10-31 16:25:15"
```

Comment:
although, aren't we making it required to have a `created_at` datetime? if we run tests where we don't have to touch anything with the `created_at` it might be annoying to have to make one just for the test to pass because it's required to have a datetime value there. what do you think?